client/src/main/java/io/obswebsocket/community/client/model/Scene.java

```diff
@@ -10,11 +10,14 @@
 @Setter
 @ToString
 public class Scene {
-
   private String sceneName;
   private Integer sceneIndex;
   private Boolean isGroup; // maybe a new feature?
 
+  public boolean getIsGroup() {
```

Comment:
The field seems to be null for non-group scenes. The example actually gave an NPE because of this.
Rather than expecting the user to do the null check I thought it would be best to have the library handle that.

Reply:
the name of the method should be `isGroup` then. Lombok sees that it is a Boolean and the get prefix is dropped

Reply:
Lombok is not smart enough to then hide the getter:
![image](https://user-images.githubusercontent.com/830783/195357852-0a566ea4-28bb-4dbb-930f-1454ca037d70.png)

From what I can tell, there are a few options:
- Rename the `getIsGroup` to `isGroup` and `@Getter(AccessLevel.NONE)` the property to hide the `Boolean` getter
- Just have both the `boolean isGroup()` and `Boolean getIsGroup()`
- Just have the `Boolean getIsGroup()` and have the user figure it out
- Or, as the UI doesn't actually seem to offer Group Scenes as an option, make the property `boolean` so that it reverts to false anyway

I'm not entirely sure why the last option wasn't originally implemented so even though it would have my preference, there might be a reason against it.

Reply:
Lombok is only smart with the primitive type, indeed

Reply:
Done